README.md

```diff
@@ -132,6 +132,18 @@ $ cargo run
 ```
 in this repository.
 
+> **alternative method:** install the tool locally
```

Comment:
nit, take it or leave it: how about specifically noting 
```suggestion
> **alternative method:** install `tokio-console` from crates.io:
```

Reply:
also, i might consider changing the "running the console" README section to present installing the CLI from crates.io _first_, as the primary suggested way to run the console, and then suggest running from a local checkout of the repository as an alternative approach for console development.

this README was written while the project was in early development and hadn't been published to crates.io yet, so we suggested running from a local git checkout as it was the only approach available at the time. now that the console is published, it would make sense to change this.